### ISSUE TITLE
CSV import broken

### ISSUE THREAD

**Issue by almereyda** [modoboa/modoboa#1430]
# Impacted versions

* Modoboa: 10.10.2
* installer used: Yes
* Webserver: Nginx

# Steps to reproduce

Run an email import from a CSV formed after http://modoboa.readthedocs.io/en/latest/moving.html#using-csv-files, i.e. one that worked before.

# Current behavior

- In the web frontend, the spinner will just rotate indifinitely.
- On the command line, an error is thrown:

```
(env)email# python /srv/modoboa/instance/manage.py modo import accounts.csv
/srv/modoboa/env/local/lib/python2.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
  """)
Traceback (most recent call last):                                              |ETA:  --:--:--
  File "/srv/modoboa/instance/manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/subcommand/base.py", line 53, in run_from_argv
    return super(SubcommandCommand, self).run_from_argv(argv)
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/subcommand/base.py", line 84, in handle
    return command.run_from_argv(argv)
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/modoboa/admin/management/commands/subcommands/_import.py", line 85, in handle
    self._import(filename, options)
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/modoboa/admin/management/commands/subcommands/_import.py", line 57, in _import
    reader = csv.reader(f, delimiter=options["sepchar"])
  File "/srv/modoboa/env/local/lib/python2.7/site-packages/backports/csv.py", line 232, in __init__
    raise TypeError('Invalid dialect parameters')
TypeError: Invalid dialect parameters
```

It feels #1342 was a bit of a too quick shot at #1312 and may need further iterations to constrain the import a little more.

- SMTPUTF8 for mailbox names is only supported with Postfix 3
- Domain names should be converted to Punycode

Manually passing `dialect='excel'` by editing `_import.py` did not help, either.

https://github.com/modoboa/modoboa/blob/83d0564c27fa1e016f011b50f94a07278532cd6c/modoboa/admin/management/commands/subcommands/_import.py#L57

The original error may reside around https://github.com/ryanhiebert/backports.csv/blob/master/src/backports/csv.py#L230
Unfortunately the upstream package does not provide a test suite.

# Expected behavior

The import completes successfully.

**Comment by almereyda** [comment]
Thanks for the introspection.

The escaped SEPCHAR yields the very same error.

**Comment by tonioo** [comment]
@almereyda Is it still happening?

**Comment by tonioo** [comment]
@almereyda Any news?

**Comment by augustodv** [comment]
hello, i have de same error and test all but the error persist.

**Comment by Spitfireap** [comment]
Should be fixed by #3039. Please open a new ticket if it still shows up.
